index.js

```diff
@@ -174,10 +175,11 @@ module.exports = {
    */
   _getFastbootTree() {
     const appName = this._name;
+    const isModuleUnification = (typeof this.project.isModuleUnification === 'function') && this.project.isModuleUnification();
```

Comment:
just out of curiosity, does it make sense to put new code for module unification [considering it's status](https://blog.emberjs.com/2019/03/11/update-on-module-unification-and-octane.html)?

Reply:
> just out of curiosity, does it make sense to put new code for module unification [considering it's status](https://blog.emberjs.com/2019/03/11/update-on-module-unification-and-octane.html)?

Hi, This was not a new code but exiting check used in contentFor hook. I copy-pasted that here. To make it cleaner, I moved the common code of isModuleUnification to a separate method. The Goal of this PR was not change exiting behavior but clean up the code inserted as part of `app-boot` hook.

We should consider removing the check for module unification as a sperate work item.

Reply:
Totally makes sense, thank you for the note @dnalagatla!